spec/pages/admin/facilities/index.rb

```diff
@@ -1,6 +1,6 @@
 module AdminPage
   module Facilities
-    class Show < ApplicationPage
+    class Index < ApplicationPage
```

Comment:
I'm just curious, why do we want to change the name of this test page to `Index` instead of `Show`?

Reply:
Thanks for addressing this [here](https://github.com/simpledotorg/simple-server/pull/755#issuecomment-575075233) 👍